### ISSUE TITLE
green tick doesn't always appear

### ISSUE THREAD

**Issue by thopri** [NOC-MDP/issue-tracking--polpred_offshore#33]
When selecting a location from map with pin and generating time series a green tick appears to show new plot.
If time series menu is selected and point put in manually without using map. no green tick appears.

Minor UX bug but would be nice to sort for consistency.

**Comment by androdnoc** [comment]
Known issue (was interested to see how long it took to notice!) Annoying bug related to how ticks are displayed and triggered by tab movements. Need to look into further to see best options.

**Comment by thopri** [comment]
yeah just a UX issue really, as the view switches to numerical view on new prediction you know it has updated plots too.
